taca/storage/cli.py

```diff
@@ -16,13 +16,14 @@ def storage(ctx, days, run):
 @storage.command()
 @click.option('--backend', type=click.Choice(['swestore']), required=True,
               help='Long term storage backend')
+@click.option('-c','--cores', type=click.INT, help='Limit the number of cores used simultaneously')
```

Comment:
I would prefer to clarify that this is about the number of runs you are going to archive simultaneously. The user doesn't have to care about cores. The implementation is perfect, I would just change:

```
@click.option('-m','--max-runs', type=click.INT, help='Limit the number of runs to be archived simultaneously')
```

vad tror du?